mne/channels/montage.py

```diff
@@ -206,6 +206,16 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
         ch_names_ = []
         pos = []
         with open(fname) as fid:
+            # Default units are meters
+            scale_factor = 1.
+            for line in fid:
+                if 'UnitPosition' in line:
+                    units = line.split()[1]
+                    if units == 'm':
+                        break
+                    elif units == 'mm':
+                        scale_factor = 1./1000.
```

Comment:
PEP8 will likely complain about lack of space between operators here. It's not a universally accepted PEP (SciPy doesn't use it for instance) but we use it